<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordDialog</class>
 <widget class="QDialog" name="PasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Update Username/Password</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="2">
         <widget class="QPushButton" name="validate_button">
          <property name="text">
           <string>Validate</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="username_label">
          <property name="text">
           <string>Username</string>
          </property>
          <property name="buddy">
           <cstring>username_line</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="username_line"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="password_label">
          <property name="text">
           <string>Password</string>
          </property>
          <property name="buddy">
           <cstring>password_line</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="password_line"/>
        </item>
        <item row="3" column="0" rowspan="2" colspan="3">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="nu_label">
          <property name="text">
           <string>New Username</string>
          </property>
          <property name="buddy">
           <cstring>nu_line</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QLineEdit" name="nu_line">
          <property name="toolTip">
           <string>Leave blank to keep the same Username</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="np_label">
          <property name="text">
           <string>New Password</string>
          </property>
          <property name="buddy">
           <cstring>np_line</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="1" colspan="2">
         <widget class="QLineEdit" name="np_line">
          <property name="toolTip">
           <string>Leave this and the next line blank to keep the same password</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="verify_np_label">
          <property name="text">
           <string>Verify New Password</string>
          </property>
          <property name="buddy">
           <cstring>verify_np_line</cstring>
          </property>
         </widget>
        </item>
        <item row="7" column="1" colspan="2">
         <widget class="QLineEdit" name="verify_np_line">
          <property name="toolTip">
           <string>Leave this and the previous line blank to keep the same password</string>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok_button</sender>
   <signal>clicked()</signal>
   <receiver>PasswordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>PasswordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
